assignment/scripts/aboutMe.js

```diff
@@ -54,6 +80,23 @@
 //      Otherwise, log "Luck is what happens when preparation meets opportunity"
 //      You'll need to research how to use switch statements!
 
+
+switch (luckyNumber) {
+    case 1:
+        console.log('First is the worst')
+        break;
+    case 2:
+        console.log('Second is the best');
+        break;
+    case 3:
+        console.log('Third is the one with the hairy chest');
+        break;
+    default:
+        console.log('Luck is what happens when preparation meets opportunity');
+}
+
+
 // 18 -- Rewrite question 13 with a `ternary` operator. You'll need to do research!
 
+adventurous ? console.log('Adventures are great!') : console.log('How about we stay home?');
```

Comment:
The key difference between a ternary and an if-block is that a ternary is an _expression_. In other words, it outputs a value. Although this works, a better approach would be to use that output, such as by saving it to a variable.

```js
let message = adventurous ? 'Adventures are great!' : 'How about we stay home?';
console.log(message);